transmart/api/v2/data_structures.py

```diff
@@ -119,6 +122,27 @@ def pretty(self, root=None, depth=0, spacing=2):
         for child in root.get('children', {}):
             s += "\n%s" % self.pretty(child, depth + 1, spacing)
         return s
+    
+    def create_dataframe(self, root=None):
+        """
+        Create dataframe representation of tree.
```

Comment:
It suggests it creates a dataframe, but it returns a list.

Reply:
Fixed